action.yml

```diff
@@ -0,0 +1,38 @@
+name: "OWASP ZAP API Scan"
+description: "Scans the web application with the OWASP ZAP API Scan"
+branding:
+  icon: "zap"
+  color: "blue"
+inputs:
+  token:
+    description: "GitHub Token to create issues in the repository"
+    required: false
+    default: ${{ github.token }}
+  target:
+    description: "Target URL"
+    required: true
+  format:
+    description: "Format openapi or soap"
```

Comment:
Inconsistent with README file which states `openapi, soap, or graphql`

Reply:
Good catch

Reply:
thanks @Ighakamo @kingthorin! I've updated the description. I based on this article 😄 
https://www.zaproxy.org/docs/docker/api-scan/